catalogs/catalog-jdbc-mysql/src/main/java/com/datastrato/gravitino/catalog/mysql/operation/MysqlTableOperations.java

```diff
@@ -78,10 +80,16 @@ protected String generateCreateTableSql(
       String comment,
       Map<String, String> properties,
       Transform[] partitioning,
+      Distribution distribution,
       Index[] indexes) {
     if (ArrayUtils.isNotEmpty(partitioning)) {
       throw new UnsupportedOperationException("Currently we do not support Partitioning in mysql");
     }
+
+    Preconditions.checkArgument(
+        null == distribution || distribution == Distributions.NONE,
+        "Currently we do not support distribution in mysql");
```

Comment:
Is it that we don't support distribution or that MySQL doesn't support distribution?

Reply:
sorry, MySQL, SQlite don't support distribution. The error msg is confusing, I will fix it

Reply:
PG also do not support distribution. I will fix all error msg